### DISCUSSION TITLE
SVG images not listed in the crossref dialog

### DISCUSSION THREAD

**Discussion by marianoibaldi** [quarto-dev/quarto-cli#5435] [Q&A]
I'm writing a document in quarto using R Studio and using SVG files as images. When I use Ctrl + / the images already used are not listed. In addition, when I use the @fig- R Studio does not show any list of figures.

Thanks in advance

**Comment by mcanouil** [comment]
It seems to be an RStudio issue rather than an issue with Quarto CLI.

Could you try to share a process to reproduce? Also you might want to look at <https://github.com/rstudio/rstudio/issues> which feels more appropriate.

**Reply by marianoibaldi** [reply]
@mcanouil you are right. I will create a process to reproduce the error and look at the issues in RStudio. Thanks!
